pingpong/ud_pingpong.c

```diff
@@ -57,7 +57,7 @@ static void *payload;
 static size_t buffer_size;
 static size_t prefix_len;
 static size_t max_msg_size = 0;
-static int timeout = 5;
+static int timeout = 30;
```

Comment:
> Sure! The example has an option -t, to get the timeout value from the
> user. We can make that as timeout for initial setup and use 5s as default
> msg timeout.

Or keep things simple and just use timeout * 10 (or some other multiple) for the setup timeout.